File: leetcode/125/validPalindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        s = s.lower()
        tmp = ""
        for i in s:
            if i.isalnum():
                tmp += i
                
        pre = 0
        last = len(tmp)-1
        while pre < last:
            if tmp[pre] != tmp[last]:
                return False
            else:
                pre += 1
                last -=1
        
        return True
        

File: leetcode/125/validPalindrome.swift
class Solution {
    func isPalindrome(_ s: String) -> Bool {
        var tmp = ""

        for i in s{
            if i.isLetter || i.isNumber{
                tmp.append(i.lowercased())
            }
        }
        let pmt = String(tmp.reversed())
        if pmt == tmp{
            return true
        }
        return false
    }
}
